timeless/templates/auth/activate.html

```diff
@@ -1,5 +1,7 @@
 {% extends 'base.html' %}
 
+{% cache 600 %}
```

Comment:
@ammaratef45 I don't know what exactly `activate` template here. but I guess we should not cache it